test/simdat.jl

```diff
@@ -18,7 +18,7 @@ using Test
         @test nrow(dat) == 16*subj_n*item_n
         @test length(unique(dat.subj)) == 4*subj_n
         @test length(unique(dat.item)) == 2*item_n
-        @test names(dat) == [:subj, :pet, :age, :item, :cond, :time, :dv]
+        @test names(dat) == ["subj", "pet", "age", "item", "cond", "time", "dv"]
```

Comment:
FYI: `propertynames(::DataFrame)` returns `Vector{Symbol}`, if you prefer to deal with things in that form.

Reply:
Ah, thank you.  I thought that `names` should have returned a vector of symbols but you are correct, I was thinking of `propertynames`.